packages/react-cosmos-component-playground/src/components/StarryBg/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import styles from './index.less';
+
+class StarryBg extends Component {
+  render() {
+    const { children } = this.props;
+    return (
+      <div className={styles.root}>
+        <div className={styles.stars}></div>
+        {children &&
+          <div className={styles.content}>
+            {children}
+          </div>}
+      </div>
+    );
+  }
+}
+
+export default StarryBg;
```

Comment:
This kinda does the same thing as `DisplayScreen` (loads some children) but it doesn't seem to have any tests even if Jest coverage reports 100% (but that's because of the way coverage works). 

Does anything stop us from adding Jest Snapshots?